docs/spec_draft.md

```diff
@@ -582,6 +583,78 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values from one element type to another
+on `operand` tensor and produces a `result` tensor.
+
+More formally, conversion from element `A` of `operand` to element `B` of
+another type is defined as the following:
+  * Conversion of same element type with different precision:
+    * The result is rounded using any normalizing rounding-direction attributes
+      from the IEEE-754 specification (e.g. `roundTiesToEven`).
+  * Conversion from integer to:
+    * signed integer: signExtend or truncate.
+    * unsigned integer: zero-extend or truncate.
+    * boolean: `B = A != 0 ? true : false`.
+    * floating-point*: `B = (float) A`.
+    * complex*: `B = ((float) A, 0.0)`.
```

Comment:
Can we check that the same consideration with floating point target applies here?